addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/UserControlledOpenRedirectScanner.java

```diff
@@ -79,9 +83,16 @@ private void checkUserControllableLocationHeaderValue(
             return;
         }
 
-        String protocol = null;
+        String requestDomain = null;
+        try {
+            requestDomain = msg.getRequestHeader().getURI().getAuthority();
+        } catch (URIException ex) {
+            LOGGER.error(
```

Comment:
IMO this should be a warn at most, it's not an error/issue in ZAP.

Reply:
changed the logger level